lib/main.js

```diff
@@ -17,6 +17,9 @@ var E = '',
      * @returns {string}
      */
     rekey = function (key, separator) {
+        if (typeof key === 'number') {
```

Comment:
could be re-written as 
```javascript
return  (typeof key === NUMBER) ? key : (key.toUpperCase().replace(/[^A-Z0-9]/g, separator || E));
```

`NUMBER = 'number',` is defined on top as a constant. Prevents needless string construction on every call.